src/index.js

```diff
@@ -0,0 +1,36 @@
+import './styles.css';
+import display from './modules/display.js';
+import { getAPI, setAPI, createGame } from './modules/API.js';
+
+const initializeGame = async () => {
+  console.log(await createGame('roblox'));
+
+  const getScore = async () => {
+    const scores = await getAPI();
+    display(scores);
+  };
+  const form = document.querySelector('form');
+
+  form.addEventListener('submit', async (event) => {
+    event.preventDefault();
+    const name = document.querySelector('.name');
+    const score = document.querySelector('.score');
+    const playername = name.value.trim();
+    const playerscore = score.value.trim();
+    const data = {
+      user: playername,
+      score: playerscore,
+    };
+    name.value = '';
+    score.value = '';
+    await setAPI(data);
+  });
+
+  document.querySelector('.refresh').addEventListener('click', () => {
+    getScore();
+  });
+
+  getScore();
+};
+
```

Comment:
![image](https://github.com/hetrox8/Project-Learderboard/assets/93039359/986f0558-eb0c-4309-ad97-445c6670ebb2)

- [ ] It would be nice if you make this LinkedIn redirects to your account 👍 because right now it is taking us to Anshuman's account 😄 Kindly fix this to make your readme descriptive and professional 😊